### PR TITLE
Add `CLIP_SPACE_FAR` built-in description to spatial shaders

### PR DESCRIPTION
**Pull request by Chaosus**
To reflect changes from https://github.com/godotengine/godot/pull/95057

### CODE REVIEW

File: tutorials/shaders/shader_reference/spatial_shader.rst [hunk 1]
```diff
@@ -185,6 +185,10 @@ shader, this value can be used as desired.
 |                                        | (this is ``true`` in the Compatibility renderer,       |
 |                                        | ``false`` in Forward+ and Forward Mobile).             |
 +----------------------------------------+--------------------------------------------------------+
+| in bool **CLIP_SPACE_FAR**             | End of a clip space range.                             |
+|                                        | When using a Vulkan renderer, it's ``0``.              |
+|                                        | On compatibility renderer, it's ``-1``.                |
++----------------------------------------+--------------------------------------------------------+
```
Comment (tetrapod00) [inline tutorials/shaders/shader_reference/spatial_shader.rst:191]:
```suggestion
| in float **CLIP_SPACE_FAR**            | Clip space far ``z`` value.                  |
|                                        | In the Forward+ or Mobile renderers, it's ``0.0``. |
|                                        | In the Compatibility renderer, it's ``-1.0``.           |
+----------------------------------------+--------------------------------------------------------+
```
It's a `float`, not a `bool`. The ``z`` value should be specified.
Floats literals should be written with the decimal like `0.0` not `0`.
Use the names of each renderer. 

(The table formatting needs to be reformed, and apply these changes to the other two locations)

Comment (Chaosus) [reply tutorials/shaders/shader_reference/spatial_shader.rst]: ah, right, ty

File: tutorials/shaders/shader_reference/spatial_shader.rst [hunk 2]
```diff
@@ -185,6 +185,10 @@ shader, this value can be used as desired.
 |                                        | (this is ``true`` in the Compatibility renderer,       |
 |                                        | ``false`` in Forward+ and Forward Mobile).             |
 +----------------------------------------+--------------------------------------------------------+
+| in float **CLIP_SPACE_FAR**            | Clip space far ``z`` value.                            |
```
Comment (clayjohn) [inline tutorials/shaders/shader_reference/spatial_shader.rst:188]: This isn't a global built in?

Comment (tetrapod00) [reply tutorials/shaders/shader_reference/spatial_shader.rst:188]: Seems not:
godot windows editor x86_64_hJdSNfayZw

Comment (Chaosus) [reply tutorials/shaders/shader_reference/spatial_shader.rst:188]: Yeah, seems like this should be fixed - https://github.com/godotengine/godot/pull/98448

**Pull request merged**